components/Episodes.vue

```diff
@@ -44,135 +44,130 @@ const props = defineProps({
       episodes.value = response.data.included
       dataLoaded.value = true
-    }).catch(function (error) {
+    } ).catch( function ( error ) {
       // console.log(error)
       axiosSuccessful.value = false
-    })
+    } )
 }
 </script>
  
 <template>
-  <div v-if="axiosSuccessful">
+  <div v-if=" axiosSuccessful ">
     <section>
       <div class="content lg:px-8">
         <div class="grid">
           <div class="col">
-            <div v-if="dataLoaded" class="recent-episodes">
-              <div
-                v-if="props.header || props.buttonText"
-                class="col flex justify-content-between align-items-center mb-3"
-              >
-                <h3 v-if="props.header">{{ props.header }}</h3>
-                <v-flexible-link v-if="props.buttonText" raw to="/episodes">
+            <div v-if=" dataLoaded " class="recent-episodes">
+              <div v-if=" props.header || props.buttonText "
+                class="col flex justify-content-between align-items-center mb-3">
+                <h3 v-if=" props.header ">{{ props.header }}</h3>
+                <v-flexible-link v-if=" props.buttonText " raw to="/episodes">
                   <Button class="p-button-rounded p-button-sm">{{ props.buttonText }}</Button>
                 </v-flexible-link>
               </div>
               <div class="grid">
-                <div
-                  v-for="(episode, index) in episodes.slice(props.paginate ? 0 : props.startCount, rowCountCalc)"
-                  :key="index"
-                  class="col-12 md:col-6 xl:col-4 mb-6"
-                  :class="{ 'xl:hidden': !props.paginate && rowCount % 2 && index === episodes.length - 1 - props.startCount }"
-                >
-                  <v-card
-                    :image="episode.attributes['image-main'].template.replace('%s/%s/%s/%s', '%width%/%height%/c/%quality%')"
-                    :alt="episode.attributes['image-main']['alt-text']"
-                    :title="episode.attributes.title"
-                    :titleLink="`/episodes/${episode.attributes.slug}`"
-                    :eyebrow="formatDate(episode.attributes['publish-at'])"
-                    :blurb="episode.attributes.tease"
-                    :height="225"
-                    :max-width="episode.attributes['image-main'].w"
-                    :max-height="episode.attributes['image-main'].h"
-                    responsive
-                    :ratio="[4, 3]"
-                    bp="max"
-                    class="radiolab-card"
-                  >
-                    <div class="divider"></div>
-                    <play-selector :episode="episode" />
-                  </v-card>
-                </div>
+                <template
+                  v-for="(      episode, index      ) in       episodes.slice( props.paginate ? 0 : props.startCount, rowCountCalc )      ">
+                  <div :key=" index " v-if=" index <= 5 " class="col-12 md:col-6 xl:col-4 mb-6"
```

Comment:
`index <= 5` might be a little unintuitive, since it's 0-indexed, perhaps change to `index < 6` because 6 episodes will be shown either way. Also, this looks like a good place to introduce a `EPISODE_DISPLAY_COUNT` constant set to 6 which could then be referenced wherever episodes are shown?